models/mace/json_to_extxyz.py

```diff
@@ -54,7 +55,12 @@
 
             special_keys = {"uncorrected_total_energy", "force", "magmom", "stress"}
             for key in {*block} - special_keys:
-                atoms.info[key] = block[key]
+                if block[key] is not None:
+                    atoms.info[key] = block[key]
+
+            atoms.info["task_id"] = task_id
```

Comment:
Maybe throw in asserts here for safety:

```suggestion
assert task_id.startswith(("mp-", "mvc-")), f"Invalid {task_id=}"
```

Unless none of the mvc materials are in MPtrj, in which case just put `startswith("mp-")`.